userspace/falco/stats_writer.cpp

```diff
@@ -48,7 +54,10 @@ static void timer_handler(int signum)
 
 bool stats_writer::init_ticker(uint32_t interval_msec, std::string &err)
 {
+#if !defined(_WIN32)
```

Comment:
This is getting a bit hard to follow

Reply:
I would just have different versions of init_ticker for each OS.